sql/hive-thriftserver/pom.xml

```diff
@@ -77,15 +77,6 @@
       <groupId>${hive.group}</groupId>
       <artifactId>hive-beeline</artifactId>
     </dependency>
-    <!-- Explicit listing hive-contrib and hive-hcatalog-core. Otherwise the maven test fails. -->
-    <dependency>
```

Comment:
Do you need the exclusions above? I guess at some point they were important

Reply:
Sorry, I don't understand what you mean.